backend/src/api/query.rs

```diff
@@ -49,17 +49,17 @@ impl Query {
     }
 
     /// Returns a list of all events the current user has read access to
-    async fn events(context: &Context) -> ApiResult<Vec<Event>> {
+    async fn all_events(context: &Context) -> ApiResult<Vec<Event>> {
         Event::load_all(context).await
     }
 
-    /// Returns a series by its Opencast ID
+    /// Returns a series by its Opencast ID.
```

Comment:
This doesn't matter at all, but I still find it funny how this comment gets a period, but the analogous one directly below doesn't. x)

Do we have any policy about punctuation in comments? I tend to not put periods if it's only a single "sentence."

Reply:
True :D Well, I always add periods. The ones without are by you.............. :eyes: